analyzer/src/funTest/assets/projects/external/example-python-flask-expected-output.yml

```diff
@@ -1,6 +1,6 @@
 ---
 project:
-  id: "PIP::src/funTest/assets/projects/external/example-python-flask/requirements.txt:0773991e36a4c69b121cdb05146d6140347d4065"
+  id: "PIP::example-python-flask:0773991e36a4c69b121cdb05146d6140347d4065"
```

Comment:
commit: Is this commit intended to be part of this PR?

Reply:
No, sorry, dropped!

Reply:
NP.